src/cljs/rems/fields.cljs

```diff
@@ -142,24 +142,30 @@
   (.. event -target -value))
 
 (defn text-field
-  [{:keys [validation on-change] :as opts}]
+  [{:keys [validation on-change collapse-info info-text] :as opts}]
   (let [placeholder (localized (:field/placeholder opts))
         value (:field/value opts)
         optional (:field/optional opts)
         max-length (:field/max-length opts)]
     [field-wrapper opts
-     [:input.form-control {:type "text"
-                           :id (field-name opts)
-                           :name (field-name opts)
-                           :placeholder placeholder
-                           :required (not optional)
-                           :aria-invalid (when validation true)
-                           :aria-describedby (when validation
-                                               (str (field-name opts) "-error"))
-                           :max-length max-length
-                           :class (when validation "is-invalid")
-                           :value value
-                           :on-change (comp on-change event-value)}]]))
+     [:<>
+      (if collapse-info [collapse
```

Comment:
The proper construct to use is `when` if you don't have the else clause of an `if`.

Reply:
Yes, I realized it shortly after and fixed that, but thanks for commentig